bodhi/server/services/updates.py

```diff
@@ -536,6 +538,23 @@ def new_update(request):
 
             if len(releases) > 1:
                 result = dict(updates=updates)
+
+            if from_tag:
+                koji = buildsys.get_session()
+                for update in updates:
+                    # Validate that <koji_tag>-pending-signing and <koji-tag>-testing exists,
+                    # if not create them
+                    koji_tag_pending_signing = update.release.get_pending_signing_side_tag(from_tag)
```

Comment:
Nitpick here but I think it makes more sense to call this side_tag rather than koji_tag.

```suggestion
                    side_tag_pending_signing = update.release.get_pending_signing_side_tag(from_tag)
```